receiver/awscontainerinsightreceiver/receiver.go

```diff
@@ -197,7 +209,9 @@ func (acir *awsContainerInsightReceiver) Shutdown(context.Context) error {
 // collectData collects container stats from Amazon ECS Task Metadata Endpoint
 func (acir *awsContainerInsightReceiver) collectData(ctx context.Context) error {
 	var mds []pmetric.Metrics
-	if acir.cadvisor == nil && acir.k8sapiserver == nil {
```

Comment:
nit: Similar to earlier feedback, maybe something like this might be more readable?

```
isLinux := acir.k8swindows != nil
isWindows := acir.cadvisor == nil && acir.k8sapiserver == nil && acir.k8swindows == nil
```

Reply:
I will address this in future PR since this complex logic might not be required.